src/views/StoryViewer.vue

```diff
@@ -57,9 +102,9 @@ export default {
     }
   },
   updated: function() {
-    if (this.currentFrame.id < this.storyLength) {
+    if (this.currentFrame.id < this.storyLength && !this.isPaused) {
       this.playNextFrame()
-    } else if(!this.storyEnd) {
+    } else if(!this.storyEnd && !this.isPaused) {
```

Comment:
in what scenario there will be paused true case when the component update?

Reply:
Whenever the story is paused we change the progress bar active classes, `updated` gets triggered then but we don't want to set another timeout if `isPaused` is true

Reply:
We could have a parent `if(!this.isPaused)` to simplify a bit if you we think that's better